packages/simple-markdown/src/__tests__/simple-markdown.test.tsx

```diff
@@ -3163,28 +3148,22 @@ describe("simple markdown", function () {
 
         it("should parse table alignments", function () {
             /**
-             * @param {string} tableSrc
-             * @param {Array<SimpleMarkdown.TableAlignment>} expectedAligns
              */
             var validateAligns = function (
                 tableSrc,
-                expectedAligns: Array<null> | Array<string>,
+                expectedAligns: Array<TableAlignment>,
             ) {
                 var parsed = blockParse(tableSrc + "\n");
                 assert.strictEqual(parsed[0].type, "table");
                 var actualAligns = parsed[0].align;
-                validateParse(actualAligns, expectedAligns);
```

Comment:
`validateParse()` is really a glorified wrapper around an equality assertion, we were fighting the types here so I've just reverted to use... `expect()` here.